distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -0,0 +1,108 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-1"]
+= Release notes for {DTProductName} 3.1
+:context: distributed-tracing-rn-3-1
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+[id="distributed-tracing-rn_3-1_component-versions"]
+== Component versions in the {DTProductName} 3.1
+
+[options="header"]
+|===
+|Operator |Component |Version
+
+|{TempoName}
+|Tempo
+|2.3.1
+
+//for each new release, update the release number in the xref path on the following line
+|xref:../../otel/otel_rn/otel-rn-3.1.adoc[{OTELName}]
+|OpenTelemetry
+|0.93.0
+
+|{JaegerName} (deprecated)
+|Jaeger
+|1.53.0
+
+|===
+
+// Tempo section
+[id="distributed-tracing-rn_3-1_tempo-release-notes"]
+== {TempoName}
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_new-features-and-enhancements"]
+=== New features and enhancements
+
+This update introduces the following enhancements for the {TempoShortName}:
+
+* Monolithic custom resource in dev-preview support.
+* Support for cluster-wide proxy environments.
+* Support for TraceQL to Gateway component.
+
+[id="distributed-tracing-rn_3-1_tempo-release-notes_bug-fixes"]
+=== Bug fixes
+
+This update introduces the following bug fixes for the {TempoShortName}:
+
+* Before this update, when a TempoStack was created with the `monitorTab` enabled in {product-title} 4.15, the required `tempo-redmetrics-cluster-monitoring-view` ClusterRoleBinding was not created. This update resolves the issue by fixing the operator RBAC for the monitor tab when the operator is deployed in an arbitrary namespace. (link:https://issues.redhat.com/browse/TRACING-3786[TRACING-3786])
+* Before this update, when a TempoStack was created on an {product-title} cluster with only an IPV6 networking stack, the compactor and ingestor pods ran in the `CrashLoopBackOff` state, resulting in multiple errors. This update provides support for IPv6 clusters.(link:https://issues.redhat.com/browse/TRACING-3226[TRACING-3226])
```

Comment:
Same comment for `TempoStack` formatting

Reply:
I'm checking all instances of TempoStack here. Thank you.